cmd/build/v2/build.go

```diff
@@ -180,7 +180,7 @@ func (bc *OktetoBuilder) buildService(ctx context.Context, manifest *model.Manif
 }
 
 func (bc *OktetoBuilder) buildSvcFromDockerfile(ctx context.Context, manifest *model.Manifest, svcName string, options *types.BuildOptions) (string, error) {
-	oktetoLog.Information("Building image for service '%s'", svcName)
+	oktetoLog.Info("Building image for service '%s'", svcName)
```

Comment:
Use infof instead to format the string
```suggestion
	oktetoLog.Infof("Building image for service '%s'", svcName)
```